### PR TITLE
bump up CouchDB queue size to 16K

### PR DESCRIPTION
**Pull request by sjfink**

### CODE REVIEW

File: common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala [hunk 1]
```diff
@@ -68,7 +68,8 @@ class CouchDbRestClient(protocol: String, host: String, port: Int, username: Str
     // Additional queue in case all connections are busy. Should hardly ever be
     // filled in practice but can be useful, e.g., in tests starting many
     // asynchronous requests in a very short period of time.
-    private val requestQueue = Source.queue(128, OverflowStrategy.dropNew)
+    private val QUEUE_SIZE = 16* 1024;
```
Comment (rabbah) [inline common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala]: should we consider making this an ansible knob?

Comment (sjfink) [inline common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala]: seems like overkill ... really going to customize this value per deployment?

Comment (rabbah) [inline common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala]: I suppose there's a few other similar values that if we were to pull out we should do consistently. So considered and rejected at this point. I can't think of a good reason one would change this alone at deployment.

**Comment by psuter** [comment]
Merged as a48202f07b5940387ab64abee4f442679f79f862.

**Pull request closed without merging**